data/data/openstack/main.tf

```diff
@@ -59,6 +60,7 @@ module "masters" {
   root_volume_type       = var.openstack_master_root_volume_type
   server_group_name      = var.openstack_master_server_group_name
   additional_network_ids = var.openstack_additional_network_ids
+  zones                  = var.openstack_master_availibility_zones
```

Comment:
there is a typo in "availability" everywhere

Reply:
oh yeah, I still have to fix that :)